.github/workflows/tests.yml

```diff
@@ -3,8 +3,18 @@ name: Tests
 on:
     push:
         branches: [main]
+        paths-ignore:
+          - "**/*.md"
+          - ".prettierrc"
+          - ".npmrc"
+          - ".gitignore"
     pull_request:
         branches: [main]
+        paths-ignore:
+          - "**/*.md"
+          - ".prettierrc"
+          - ".npmrc"
+          - ".gitignore"
```

Comment:
There are some changes we could do to the paths:
- There are more static files we could include, e.g. .env.example, .eslintrc.cjs, .eslintignore
- We could also ignore assets such as images

Reply:
Yeah that sounds good. I will add those too...